### ISSUE TITLE
[Bug Report] Remove audio from previews and avoid fallback

### ISSUE THREAD

**Issue by alexhk** [stashapp/stash#1404]
Stash fails to generate previews on older AVI files and has to try again with the fallback method. 

The error **[mp3float @ 0000022c7b5af4c0] Header missing** is very common with older AVI files.

I wonder why audio is included in the preview files?! Removing audio saves resources and avoids errors.

The log looks like this:

```
[generator] generating sprite image for [...]input.avi 
ffmpeg error when running command <C:\Tools\ffmpeg\bin\ffmpeg.exe -v error -xerror -ss 62.33 -i [...]input.avi  -t 0.75
 -max_muxing_queue_size 1024 -y -c:v libx264 -pix_fmt yuv420p -profile:v high -level 4.2 -preset slow -crf 21 -threads 4
 -vf scale=640:-2 -c:a aac -b:a 128k -strict -2 generated\tmp\preview_053f1351a064c3e2_001.mp4>:
 [mp3float @ 0000022c7b5af4c0] Header missing
[generator] failed generating scene preview, trying fallback 
```

In our case the error is usually shown twice in ffmpeg's output: once when it checks the mapping (it will continue) and again when it does the decoding (now it will exit due to `-xerror`).

If you use `-map 0` or alternatively `-an` in order to ignore audio it will work.

I am actually not sure why you would want to decode/encode audio into the previews. Maybe I am missing something, it has been a few long days with little sleep.

**Comment by WithoutPants** [comment]
We have a setting in Interface that enables sound when previewing a video. Making this a configurable option is worth doing though.

**Comment by WithoutPants** [comment]
Fixed by #1454
